test/functionalTest/cases/3867_textUnrestricted_should_be_recursively_applied_to_JSON_objects_and_array/textUnrestricted_should_be_recursively_applied_to_JSON_objects_and_array.test

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+TextUnrestricted should be recursively applied to JSON objects and array
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v2/entities with attribute type TextUnrestricted
```

Comment:
I'd suggest to add a second step to retrieve the entity, i.e:

```suggestion
# 01. POST /v2/entities with attribute type TextUnrestricted
# 02. GET /v2/entities to check the TextUnrestricted fields are correctly rendered
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3919/commits/4e3c1658dea590ac07c9e7e2faa5c7a233d41afd